docs/platform/release-notes/tizen-7-0-m1.md

```diff
@@ -0,0 +1,545 @@
+  - An operation of the font description has been optimized.
+  - An unused signal of the StyleManager has been disconnected with Controls.
+- Open source
+  - Harfbuzz version 2.6.7 has been upgraded to version 3.4.0.
+  - SDL version 2.0.8 has been upgraded to version 2.0.14.
+  - AT-SPI2-Core version 2.39.90 has been upgraded to version 2.42.0.
+
+#### Fixes
+
+- The SceneHolder's RenderTarget initialization defect has been fixed.
+- Various partial update defects have been fixed.
+- Various ATSPI defects have been fixed.
+- Various text related defects have been fixed.
+
+
+### Multimedia framework
+
+#### New and changed features
+
+- Open source
+  - GStreamer
+    - Version 1.16.2 has been upgraded to version 1.20.0.
+    - mms plugin has been removed.
+  - libnice
+    - Version 0.1.17 has been upgraded to version 0.1.18.
+  - OpenCV
+    - Version 4.5.0 has been upgraded to version 4.5.3.
+  - Pulseaudio
+    - Version 13.0 has been upgraded to version 15.0.
+  - libsndfile
+    - Version 1.0.28 has been upgraded to version 1.0.31.
+  - ffmpeg
+    - Version 4.3.1 has been upgraded to version 4.4.
+  - libtiff
+    - Version 4.1.0 has been upgraded to version 4.3.0.
+  - GraphicsMagick
+    - Version 1.3.35 has been upgraded to version 1.3.36.
+  - libwebp
+    - Version 1.1.0 has been upgraded to version 1.2.1.
+  - taglib
+    - Version 1.11.1 has been upgraded to version 1.12.
+  - libjpeg-turbo
+    - Version 2.0.6 has been upgraded to version 2.1.1.
+  - lcms2
+    - Version 2.4 has been upgraded to version 2.12.
+- Native WebRTC
+  - Statistics API set has been added.
+  - Data channel buffered amount API set has been added.
+  - Frame rate set/get functions for video source have been added.
+  - Bundle policy API has been added.
+  - Support for in-band FEC has been added.
+- Image Util
+  - Support for Jpeg-XL has been added.
+    - Open source libjxl 0.6.1 version has been applied.
+    - Support for decoding Jpeg-XL image format has been added.
+    - Support for encoding Jpeg-XL image format has been added.
+  - The image_util_encode_set_webp_lossless API has been deprecated.
+- Media Editing Framework
+  - Support for the most frequently used editing functionalities has been added.
+    - Place audio/video/image contents freely on timeline.
+    - Cut and paste clip and add video transition for overlapped video.
+    - Add background music and adjust volume level by each audio track.
+    - Add effects using various GStreamer effect plugins.
+  - Pause & resume, share editing artifacts
+
+#### Fixes
+
+- Native WebRTC
+  - Various types of memory leak defects have been fixed.
+  - A crash when making packet lost custom event has been fixed.
+  - ‘ice-gathering-state’ signal defect regarding data channel has been fixed.
+
+
+### Network and Connectivity
+
+#### New and changed features
+
+- Matter
+  - Matter which aims for unifying and IP-based connectivity protocol has been added.

```

Comment:
A bit confused by the sentence, so provided this suggestion.
```suggestion
  - Matter which aims to unify an IP-based connectivity protocol has been added.

```

Reply:
will accept